docs/developers/README.md

```diff
@@ -22,7 +22,12 @@ Latest builds are available for testing at:
 
 # Browser support
 
-Chart.js offers support for all browsers where canvas is supported.
+Chart.js offers support for the following browsers:
+* Chrome
+* Firefox
+* Internet Explorer 11
+* Edge
+* Safari
 
 Browser support for the canvas element is available in all modern & major mobile browsers. [CanIUse](http://caniuse.com/#feat=canvas)
```

Comment:
do you still want this line since you removed the mention of canvas above?

Reply:
I thought about that. Wasn't sure if it was best to leave it in or not. I'm happy to change it

Reply:
What about combining the lines into:

Chart.js works in [any browser with canvas support](http://caniuse.com/#feat=canvas). This is any modern & major browser including:
* Chrome
* Firefox
* Internet Explorer 11
* Edge
* Safari

Reply:
Actually, I take it back. I would remove any mention of canvas. Canvas is supported in IE9 and above, but we only support IE11+, so it's not accurate to say that we support browsers that support canvas.

Reply:
Agree, we should remove the part about canvas compatibility since we use other features that are not supported in all browsers. Ideally, we should also decide about minimal versions for all browsers, not only IE 11+.

Reply:
I went ahead and pick some min versions for all browsers. I started by picking the Firefox ESR release (45) and then pick similar versions for other browsers that were out around the same time.

Reply:
I wouldn't think it's necessary to pick minimum versions for the evergreen browsers. For a browser that updates itself automatically like Chrome I think we should just support the latest version

Reply:
@benmccann maybe you are right, I'm not sure about policies in companies that may want to lock a specific browser version (if possible). I guess it doesn't hurt to list minimal supported versions for which we are sure Chart.js is compatible with.

@etimberg looks good, thanks!

Reply:
Part of the issue I was thinking is that it's really hard for us as contributors to test old versions of Chrome, etc. I'm not sure how I'd do that and would really prefer not to have to.

Reply:
I think us testing the latest versions is fine, but we also don't want to say 'Chrome' and have some using an ancient version that doesn't work. I was thinking we could roll these supported version as the 2nd oldest Firefox ESR and then whatever Chrome version was available then. How does that sound?

Reply:
I don't know that we'd come back to update these versions whenever new browser versions are released. What if we said:

* Chrome stable
* Firefox stable and ESR
* Internet Explorer 11
* Edge stable and other most recent supported version
* Safari stable

Reply:
I'm ok with that. @simonbrunel are you?

Reply:
This is a commitment with our users (not - only - contributors) that we will make sure it doesn't break for the listed versions and if it breaks, that will be considered as a bug and not a "won't fix, because it works in the latest stable version". Being vague on this commitment will not make Chart.js more trustful.

Having precise supported versions actually help us to be consistent and prevent debate on PR introducing new browser features: let's imagine we are not supporting IE11 anymore, then what features I'm allow to use in my contributions? Is it ok to break all versions before "stable" as soon it's releases?

I'm fine for testing only stable versions when contributing until we have an automated way to test all supported browsers. Anyway, with versions picked by @etimberg, we should be guarantee that if it breaks for Chrome/Firefox, it will break for IE11 first. At this point, since IE11 is the one that defines the features scope, I prefer to commit on locked versions for other browsers as you did in your last changes.

https://www.highcharts.com/docs/getting-started/compatibility

Reply:
Yes, I agree that it's a commitment primarily with our users.

However, I don't think it's vague to say that we will support only the latest version of Chrome. I think that's very clear.

In the case of dropping IE11, what I'm suggesting is that we would support the latest version of Edge and the previous stable version. That would be Edge 38.14393 released over 1 year ago.

What I'm wary of is making a commitment to our users that we don't have a strong way of keeping. If we can't test supported browsers then how do we know we aren't breaking them?